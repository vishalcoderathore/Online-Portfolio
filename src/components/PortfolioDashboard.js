import React from 'react';
import Navbar from './Navbar';
import PortfolioIntro from './PortfolioIntro';
import PortfolioEducation from './PortfolioEducation';

const PortfolioDashboard = () => (
    <div>
        <Navbar />
        <PortfolioIntro />
        <PortfolioEducation />
    </div>
);

export default PortfolioDashboard;