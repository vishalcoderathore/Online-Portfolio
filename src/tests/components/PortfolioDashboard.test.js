import React from 'react';
import PortfolioDashboard from '../../components/PortfolioDashboard';
import { shallow } from 'enzyme';

test('should render portfolio dashboard page correctly', () => {
    const wrapper = shallow(<PortfolioDashboard />);
    expect(wrapper).toMatchSnapshot();
});