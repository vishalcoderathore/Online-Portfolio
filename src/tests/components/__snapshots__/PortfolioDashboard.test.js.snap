// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render portfolio dashboard page correctly 1`] = `
<div>
  <Header />
  <PortfolioIntro />
  <PortfolioEducation />
</div>
`;
